packages/commonwealth/client/scripts/App.tsx

```diff
@@ -9,13 +9,13 @@ import { RouterProvider } from 'react-router-dom';
 import { ToastContainer } from 'react-toastify';
 import { queryClient } from 'state/api/config';
 import { featureFlags } from './helpers/feature-flags';
-import { CWIcon } from './views/components/component_kit/cw_icons/cw_icon';
+import CWLoadingSpinner from './views/components/component_kit/new_designs/CWLoadingSpinner';
```

Comment:
some time ago we had a conversation with design team about spinners nomenclature and there was a suggestion from design team to change the name of our components:
- CWLoadingSpinner => CWCircleMultiplySpinner
- CWCircleSpinner => CWCircleRingSpinner

Do you mind changing this in this PR. Or creating another ticket for this? [See context](https://commonxyz.slack.com/archives/C052DMYFKDL/p1706722952215969?thread_ts=1706721213.866819&cid=C052DMYFKDL)

Reply:
@masvelio 
Created a card around those two changes and tagged you in them.